src/index.js

```diff
@@ -1,4 +1,5 @@
 import {spawn} from 'cross-spawn';
+import commandConvert from './command';
```

Comment:
This doesn't appear to be in the npm package dist artifacts 
![image](https://cloud.githubusercontent.com/assets/762949/19084996/7f9315ca-8a1d-11e6-994b-477b433ec62d.png)